regression/cbmc-java/address_space_size_limit1/test.desc

```diff
@@ -0,0 +1,5 @@
+CORE
+Test.class
+--object-bits 4
```

Comment:
CBMC crashes when `--object-bits` is set to 64 or higher, or to 0. If there is a hard-coded limit, then it should be guarded by the command line parser, and there should be a regression test that checks that cbmc exits gracefully.